azkaban-common/src/main/java/azkaban/metrics/ContainerizationMetricsImpl.java

```diff
@@ -127,6 +129,12 @@ public void markPodAppFailure() {
   @Override
   public void markExecutionStopped() { executionStopped.mark(); }
 
+  @Override
+  public void markOOMKilled() { executionStopped.mark(); }
```

Comment:
These 2 are identical now, any plan to improve granularity? e.g. for OOMKilled, mark exec-stop as well as a new metrics `oomKilled.mark()`?

Reply:
No plan yet. There will be a huge amount of reasons why a pod moves to Execution_stopped. It is beyond the scope of this pr to improve granularity here